easyocr/easyocr.py

```diff
@@ -423,8 +444,522 @@ def recognize(self, img_cv_grey, horizontal_list=None, free_list=None,\
+                    recog_network = 'generation2'
+                elif 'ja' in lang_list:
+                    self.setModelLanguage('japanese', lang_list, ['ja','en'], '["ja","en"]')
+                    model = recognition_models['gen2']['japanese_g2']
+                    recog_network = 'generation2'
+                elif 'ko' in lang_list:
+                    self.setModelLanguage('korean', lang_list, ['ko','en'], '["ko","en"]')
+                    model = recognition_models['gen2']['korean_g2']
+                    recog_network = 'generation2'
+                elif 'ta' in lang_list:
+                    self.setModelLanguage('tamil', lang_list, ['ta','en'], '["ta","en"]')
+                    model = recognition_models['gen1']['tamil_g1']
+                    recog_network = 'generation1'
+                elif 'te' in lang_list:
+                    self.setModelLanguage('telugu', lang_list, ['te','en'], '["te","en"]')
+                    model = recognition_models['gen2']['telugu_g2']
+                    recog_network = 'generation2'
+                elif 'kn' in lang_list:
+                    self.setModelLanguage('kannada', lang_list, ['kn','en'], '["kn","en"]')
+                    model = recognition_models['gen2']['kannada_g2']
+                    recog_network = 'generation2'
+                elif set(lang_list) & set(bengali_lang_list):
+                    self.setModelLanguage('bengali', lang_list, bengali_lang_list+['en'], '["bn","as","en"]')
+                    model = recognition_models['gen1']['bengali_g1']
+                    recog_network = 'generation1'
+                elif set(lang_list) & set(arabic_lang_list):
+                    self.setModelLanguage('arabic', lang_list, arabic_lang_list+['en'], '["ar","fa","ur","ug","en"]')
+                    model = recognition_models['gen1']['arabic_g1']
+                    recog_network = 'generation1'
+                elif set(lang_list) & set(devanagari_lang_list):
+                    self.setModelLanguage('devanagari', lang_list, devanagari_lang_list+['en'], '["hi","mr","ne","en"]')
+                    model = recognition_models['gen1']['devanagari_g1']
+                    recog_network = 'generation1'
+                elif set(lang_list) & set(cyrillic_lang_list):
+                    self.setModelLanguage('cyrillic', lang_list, cyrillic_lang_list+['en'],
+                                          '["ru","rs_cyrillic","be","bg","uk","mn","en"]')
+                    model = recognition_models['gen2']['cyrillic_g2']
+                    recog_network = 'generation2'
+                else:
+                    self.model_lang = 'latin'
+                    model = recognition_models['gen2']['latin_g2']
+                    recog_network = 'generation2'
+            self.character = model['characters']
+
+            # model_path = os.path.join(self.model_storage_directory, model['filename'])
+            # check recognition model file
```

Comment:
delete this and raise exception with message "pretrained model loading not supported"